src/App.css

```diff
@@ -0,0 +1,79 @@
+.calculatorSection {
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  margin: 8rem auto;
+}
+
+.calculatorSection h1 {
+  font-size: 3rem;
+  font-weight: 600;
+  font-style: oblique;
+  border-bottom: 5px solid #f5913e;
+}
+
+.calc {
+  width: 40%;
+  font-size: 20px;
+}
+
+.navbar {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: #232858;
+  padding: 15px 10px;
+  color: white;
+  font-weight: 600;
+}
+
+.navbar ul {
+  list-style: none;
+  display: flex;
+}
+
+.navbar ul > li {
+  padding: 5px 10px;
+  font-size: 20px;
+  border-right: 1px solid white;
+}
+
+.navbar ul > li:last-child {
+  border-right: none;
+}
+
+.navbar a {
+  color: white;
+  text-decoration: none;
+  font-weight: 600;
+  letter-spacing: 2px;
+}
+
+.container {
+  text-align: center;
+  margin: 5rem auto;
+}
+
+.container h1 {
+  font-weight: 600;
+  letter-spacing: 1px;
+  font-style: italic;
+  font-size: 3rem;
+}
+
+.container p {
+  margin: 3rem 10rem 3rem 3rem;
+  letter-spacing: 1px;
+  background: #e9d5d5;
+  padding: 2rem;
+  border-radius: 2rem;
+  line-height: 1.6rem;
+}
+
+.container h3 {
+  background: #e9d5d5;
+  padding: 1.5rem;
+  margin: 3rem 57rem 3rem 2rem;
```

Comment:
- [ ] I think the right margin is unreasonably big and it affects the component display. Kindly make changes here. :pray: 

![image](https://user-images.githubusercontent.com/64894927/135511335-97a1c9ed-c2c1-4ecb-af8f-e638661ddf22.png)